rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -415,24 +423,34 @@ def concat_episode(self, episode_chunk: "_Episode"):
         assert self.t == episode_chunk.t_started
         # Pop out our end.
         self.observations.pop()
+        self.infos.pop()
 
         # Extend ourselves. In case, episode_chunk is already terminated (and numpyfied)
         # we need to convert to lists (as we are ourselves still filling up lists).
         self.observations.extend(list(episode_chunk.observations))
         self.actions.extend(list(episode_chunk.actions))
         self.rewards.extend(list(episode_chunk.rewards))
+        self.infos.extend(list(episode_chunk.infos))
         self.t = episode_chunk.t
         self.states = episode_chunk.states
 
         if episode_chunk.is_terminated:
             self.is_terminated = True
         elif episode_chunk.is_truncated:
             self.is_truncated = True
+
+        for k, v in episode_chunk.extra_model_outputs.items():
+            self.extra_model_outputs[k].extend(list(v))
         # Validate.
         self.validate()
 
     def add_initial_observation(
-        self, *, initial_observation, initial_state=None, initial_render_image=None
+        self,
+        *,
+        initial_observation,
+        initial_info,
```

Comment:
Let's make the default for `initial_info` also None, just in case users really don't care about these and never pass them around.
If None, further down in the code, we'll just automatically append an empty dict, then.